### PR TITLE
FEAT: Improve extensions quality

### LINKED ISSUE #5545: Update extension style
### Description of the current limitations

Choke designer has added a new style for extensions. Previous extensions should have the same style.

### Code sample expected

_No response_

### Useful links and references

_No response_

### PR DESCRIPTION
**Pull request by Samuelopez-ansys**
## Description
Add documentation of extensions
Unify UI style

## Issue linked
Close #5545 

## Checklist
- [x] I have tested my changes locally.
- [x] I have added necessary documentation or updated existing documentation.
- [x] I have followed the coding style guidelines of this project.
- [x] I have added appropriate tests (unit, integration, system).
- [x] I have reviewed my changes before submitting this pull request.
- [x] I have linked the issue or issues that are solved by the PR if any.
- [x] I have agreed with the Contributor License Agreement ([CLA](https://developer.ansys.com/form/cla-acceptance)).

### CODE REVIEW

File: doc/source/User_guide/extensions.rst [hunk 1]
```diff
@@ -112,6 +140,21 @@ They are small automated workflow with a simple UI.
             Import a CSV file containing sources layout and power dissipation information.
 
 
+Circuit extensions
+~~~~~~~~~~~~~~~~~~
+
+Pre-installed extensions are available at Circuit level.
+They are small automated workflow with a simple UI.
```
Comment (MaxJPRey) [inline doc/source/User_guide/extensions.rst:147]:
```suggestion
They are small automated workflows with a simple UI.
```

Comment (SMoraisAnsys) [reply doc/source/User_guide/extensions.rst:147]: I was going to ask for the same change, why is this solved but no changes performed ?

**Review by SMoraisAnsys — APPROVED** [review]
LGTM, thanks for improving the documentation and unifying the extensions. Also, I like the added checks in the workflows !

**Comment by Samuelopez-ansys** [comment]
I moved it to draft because the same had to be applied to more files, and I am still fixing the other extensions. 

I have also uncommented the unit tests that were failing due to PyEDB issues, and now they are passing :) so we can improve the coverage.

**Comment by SMoraisAnsys** [comment]
> I moved it to draft because the same had to be applied to more files, and I am still fixing the other extensions.
> 
> I have also uncommented the unit tests that were failing due to PyEDB issues, and now they are passing :) so we can improve the coverage.

I think this problem were related to the dotnet issues that was fixed two weeks ago. I'll add the same changes in PyAEDT.

File: src/ansys/aedt/core/workflows/circuit/import_schematic.py [hunk 2]
```diff
@@ -38,26 +39,27 @@
...
 extension_arguments = {"asc_file": ""}
-extension_description = "Import schematic to Circuit."
+extension_description = "Import schematic to Circuit"
 
 
 def frontend():  # pragma: no cover
-
     import tkinter
     from tkinter import filedialog
     from tkinter import ttk
```
Comment (gmalinve) [inline src/ansys/aedt/core/workflows/circuit/import_schematic.py:48]:
```suggestion
    import tkinter as tk
    from tkinter import filedialog
    import tkinter.ttk as ttk
```
@Samuelopez-ansys @SMoraisAnsys for me this is better. What do you think?

File: src/ansys/aedt/core/workflows/circuit/import_schematic.py [hunk 3]
```diff
@@ -38,26 +39,27 @@
...
 
     import PIL.Image
     import PIL.ImageTk
+    from ansys.aedt.core.workflows.misc import ExtensionTheme
 
     master = tkinter.Tk()
-
-    master.geometry("750x250")
-
+    master.geometry("650x150")
```
Comment (gmalinve) [inline src/ansys/aedt/core/workflows/circuit/import_schematic.py:55]: @Samuelopez-ansys a small comment. I think that if you don't pass the size, tkinter will automatically fit the content in the window.

File: doc/source/User_guide/pyaedt_extensions_doc/hfss/push_excitation.rst [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+or it can be used in batch mode via command line arguments.
+
+Features
+--------
+
+- Automatically detects and lists available ports in the active HFSS design.
+- Allows users to browse and select a time-domain excitation file.
+- Supports both light and dark themes for the GUI.
+- Assigns excitations programmatically using a batch-mode interface.
+- Validates file paths and port selections to ensure proper configuration.
```
Comment (gmalinve) [inline doc/source/User_guide/pyaedt_extensions_doc/hfss/push_excitation.rst:17]: @Samuelopez-ansys for me here (and in all the other files) shouldn't be in third person.

File: src/ansys/aedt/core/workflows/circuit/import_schematic.py (hunk 2 above)
Comment (Samuelopez-ansys) [reply src/ansys/aedt/core/workflows/circuit/import_schematic.py:48]: Let's do it in a different PR

File: doc/source/User_guide/pyaedt_extensions_doc/hfss/push_excitation.rst (hunk 4 above)
Comment (Samuelopez-ansys) [reply doc/source/User_guide/pyaedt_extensions_doc/hfss/push_excitation.rst:17]: Why? They are the extension features.

**Review by gmalinve — APPROVED** [review]

**Pull request merged**